src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -406,6 +409,10 @@ def _process_heatmap_tools(heatmap_tools: set[str],
         _create_import_files(root_dir, rows_output, heatmap_rows_list)
         _create_import_files(root_dir, cell_details_output, heatmap_cell_details_list)
 
+        result_dir = _locate_dir(root_dir, kbase_collection, load_ver, tool='microtrait')
```

Comment:
The tool isn't necessarily microtrait, we could add more heatmap tools in the future

Reply:
:+1: